packages/core/src/Modifier.ts

```diff
@@ -8,6 +8,7 @@ export interface Modifier {
     constructor: ModifierConstructor & this;
 }
 export class Modifier {
+    public preserve: boolean = true;
```

Comment:
`public` is the default, we have chosen to omit it when defining class properties. The only place where we use it is in constructor parameters.

Reply:
I'll rewrite the commit ;)